catalog/service/rest/src/main/java/org/projectnessie/catalog/service/rest/IcebergConfigurer.java

```diff
@@ -305,17 +307,28 @@ private Map<String, String> adlsConfigOverrides(StorageUri warehouseLocation) {
     AdlsFileSystemOptions fileSystemOptions =
         adlsOptions.effectiveOptionsForFileSystem(fileSystem, secretsProvider);
     // FIXME send account key and token?
-    fileSystemOptions
-        .account()
-        .ifPresent(
-            key -> {
-              configOverrides.put(ADLS_SHARED_KEY_ACCOUNT_NAME, key.name());
-              configOverrides.put(ADLS_SHARED_KEY_ACCOUNT_KEY, key.secret());
-            });
-    fileSystemOptions
-        .sasToken()
-        .ifPresent(
-            s -> configOverrides.put(ADLS_SAS_TOKEN_PREFIX + location.storageAccount(), s.key()));
+    AdlsFileSystemOptions.AzureAuthType authType =
```

Comment:
Heads up: this will conflict w/ #9100

Reply:
Thanks! I removed the credentials sending part.